core/unit_test/CMakeLists.txt

```diff
@@ -649,7 +649,10 @@ FUNCTION (KOKKOS_ADD_INCREMENTAL_TEST DEVICE)
     SOURCES ${${DEVICE}_SOURCES}
   )
 
-  TARGET_INCLUDE_DIRECTORIES( ${PACKAGE_NAME}_IncrementalTest_${DEVICE} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/incremental )
+  SET(EXE_NAME ${PACKAGE_NAME}_IncrementalTest_${DEVICE})
+  IF(TARGET ${EXE_NAME})
```

Comment:
```suggestion
  // Check that the target was actually created because in a TribITS build
  // where only tests marked as PERFORMANCE enabled it would not be.
  IF(TARGET ${EXE_NAME})
```